.github/workflows/build.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.11"
```

Comment:
This should be 3.9 now if i'm not mistaken.
And yes, I would stick to a python version not selecting "latest". I have been bitten too often by open ended dependencies.

Reply:
It's just the template CI itself so it doesn't matter so much what we pick, I kinda want to stick with 3.11

Reply:
"just". In away these are test runs for the templated workflows in the plugins. And you chose python 3.9 as a default there. Actually I would even be ok to hardcode the value of "python_version" in the `plugin-template` binary. There is really no use (anymore) for the plugin maintainer to have a choice.

It used to be the python version pulp was running on when we used the ansible based installer, but it lost that purpose a long time ago.

Reply:
Would you have any objection to using 3.11 for everything other than what Pulp itself is running on?

Reply:
No, that's ok. And I actually believe the plugin template should be in charge, because it also provides all the code that is run.

Reply:
Should `python_version` be fully deprecated then?

Reply:
I'd keep it in the templates and hard code in the application code so we have a single place to change it.

I think the python version comes with the image we allow to select (In an even more flexible way). I wouldn't even see a good way to assert the version pulp runs on is the one specified. And so the existence of the option raises false expectations. I may be missing your point though.

Reply:
It's provided by the image we chose currently, though I know we've had various requests around making the Python version variable, and I just wasn't sure how exactly that would be implemented

But honestly that's out of scope so I'm going to drop that line of discussion